plugins/sharding/shard_executor_base.rb

```diff
@@ -0,0 +1,211 @@
+module Groonga
+  module Sharding
+    class ShardExecutorBase
```

Comment:
How about `StreamShardExecutor` or something?

Reply:
Sounds good.

I have renamed it.